code/src/App.js

```diff
@@ -1,9 +1,52 @@
-import React from 'react';
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { SubmitThoughts } from 'components/SubmitThoughts';
+import { ListThoughts } from 'components/ListThoughts';
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([])
+  const [submitThoughts, setSubmitThoughts] = useState('')
+  const [loading, setLoading] = useState(false)
+
+  const fetchThoughts = () => {
+    setLoading(true)
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts')
+      .then((response) => response.json())
+      .then((data) => setThoughtsList(data))
+      .catch((error) => console.log(error))
+      .finally(() => { setLoading(false) })
+  }
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault()
+    setLoading(true)
+
```

Comment:
Since the "handleFormSubmit" handles the submission of a new thought it could be moved into your component "submitThoughts.js" instead if we want to continue on "cleaning" the app.js a bit.